Added documentation for adding a TSC Member to the Release Manager (#180)
fairpm/tsc: process/release-manager.md

- **Release Managers** handle the operational tasks of coordinating and executing releases

These roles may overlap:
- A TSC Member may also serve as a Release Manager
- A Release Manager does not need to be a TSC Member
- Release Managers report to the TSC on release status and process improvements

## Responsibilities

Release Managers are responsible for:

- **Release Coordination**: Coordinating release activities across working groups and projects
- **Quality Assurance**: Ensuring all pre-release checks and tests are completed
- **Artifact Management**: Building, signing, and publishing release artifacts
- **Documentation**: Maintaining release notes, changelogs, and upgrade guides
- **Communication**: Announcing releases and coordinating with the community
- **Process Improvement**: Identifying and implementing improvements to the release process
- **Compliance**: Ensuring releases meet security, licensing, and regulatory requirements

## Required Permissions

To perform their duties, Release Managers require the following permissions:

### GitHub Permissions

> - **Write access** to all project repositories under the FAIR organization

Reviewer: ```suggestion
- **Write access** or higher to the release source repository
```

- **Maintainer role** on GitHub teams managing release repositories
- **Admin access** to GitHub Actions and Workflows for CI/CD pipelines
- **Permission to create releases** and publish packages
- **Access to GitHub Packages** for publishing artifacts
- **Permission to manage tags** and protected branches

> - **Access to repository secrets** needed for signing and publishing

Reviewer: This is not a current requirement

> 

Reviewer: ## Managing the **Release Managers** team

A Release Managers team exists at the organization level. This team has `maintain` access to the `fairpm/tsc` repository.

### Add a Release Manager:

1. Go to the [Release Managers team](https://github.com/orgs/fairpm/teams/release-managers)
2. Click the *Add a member* button
3. Search for the user you wish to add to the team
4. Click the + sign next to the user and then click the **Invite** button

### Remove a Release Manager:
1. Go to the [Release Managers team](https://github.com/orgs/fairpm/teams/release-managers)
2. Select the member you wish to remove
3. Click **Remove from team**
